app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt

```diff
@@ -225,12 +228,12 @@ private fun WallpaperThumbnailItem(
             .then(border)
             .clickable { onSelect(wallpaper) }
     ) {
-        if (wallpaper != defaultWallpaper) {
-            val contentDescription = stringResource(
-                R.string.wallpapers_item_name_content_description, wallpaper.name
-            )
+        val contentDescription = stringResource(
```

Comment:
Perhaps inline the contentDescription now that it's not conditional

Reply:
Updated!